File: corpus/py/C;0670.py
static final 
int tam,k = 20,i
print("Digite o tamanho do vetor")
tam = int(input())
for i=0 in range(0, tam):
    print("Digite o valor da posição"+i)
CriarMatriz(vetor,tam)
ImprimirMatriz(matriz,tam)
def CriarMatriz(int v[], int tam):
    int i,j
    for i=0 in range(0, tam):
        for j=0 in range(0, tam):
            if i==j:
            else:
                print("Digite o valor da linha "+i +" e coluna " +j)
    return matriz[tam][tam]
def ImprimirMatriz(int m[][], int tam):
    int i,j
    for i=0 in range(0, tam):
        for j=0 in range(0, tam):
            print("Digite o valor da matriz"+m[i][j])

File: corpus/2018q1/_p2q3_ok/_check/C;0670.java

import java.util.Scanner;

public class Questao3 {
static final Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        int tam,k = 20,i;
        System.out.println("Digite o tamanho do vetor");
        tam = input.nextInt();
        int vetor[] = new int[tam];
        int matriz[][] = new int[tam][tam];
        for (i=0; i<tam; i++){
            System.out.println("Digite o valor da posição"+i);
            vetor[i]= input.nextInt();
        }
        CriarMatriz(vetor,tam);
        ImprimirMatriz(matriz,tam);
    }
    public static int CriarMatriz(int v[], int tam){
        int matriz[][] = new int[tam][tam];
        int i,j;
         for (i=0; i<tam; i++){
             for (j=0; j<tam; j++){
                 if (i==j){
                      matriz[i][j] = v[i];
                 }
                 else{
                System.out.println("Digite o valor da linha "+i+" e coluna " +j);
                 matriz[i][j] = input.nextInt();
              }  
             }
}
         return matriz[tam][tam];
    }
    public static void ImprimirMatriz(int m[][], int tam){
        int i,j;
         for (i=0; i<tam; i++){
             for (j=0; j<tam; j++){
                 System.out.println("Digite o valor da matriz"+m[i][j]);

    }
}
    }
}
